### PR TITLE
III-4975 PSR-11 labels

### PR DESCRIPTION
**Pull request by JonasVHG**
### Changed

- `LabelServiceProvider` is now shared via the Container instead of the Silex App.
- `LabelServiceProvider` is moved outside the Silex folder.

---
Ticket: https://jira.uitdatabank.be/browse/III-4975

### CODE REVIEW

File: app/Labels/LabelServiceProvider.php [hunk 1]
```diff
@@ -0,0 +1,360 @@
...
+            }
+        );
+    }
+
+    private function setUpReadModels(Container $container): void
+    {
+        $container->addShared(
+            self::JSON_READ_REPOSITORY,
+            function () use ($container): ReadRepositoryInterface {
+                $labels = file_exists(__DIR__ . '/../../../config.excluded_labels.php') ? require __DIR__ . '/../../../config.excluded_labels.php' : [];
```
Comment (bertramakers) [inline app/Labels/LabelServiceProvider.php:158]:
```suggestion
                $labels = file_exists(__DIR__ . '/../../config.excluded_labels.php') ? require __DIR__ . '/../../config.excluded_labels.php' : [];
```
File location has changed so this path needs to change as well

**Review by bertramakers — CHANGES_REQUESTED** [review]

**Comment by bertramakers** [comment]
Other changes look good! I'd like to get rid of all these constants for the service names though, but we'll do that later in the general cleanup of service names to use FQN's as much as possible

**Review by bertramakers — APPROVED** [review]
Perfect now! 👌

**Comment by bertramakers** [comment]
@JonasVHG remember to merge this. There are some merge conflicts now

**Comment by JonasVHG** [comment]
> @JonasVHG remember to merge this. There are some merge conflicts now



> @JonasVHG remember to merge this. There are some merge conflicts now

Sorry, I've overlooked this a bit with my many meetings yesterday. I've finished the merge and am running the acceptance tests atm.

**Pull request merged**